go.mod

```diff
@@ -47,22 +48,36 @@ require (
 	github.com/prometheus/common v0.44.0 // indirect
 	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/net v0.17.0 // indirect
-	golang.org/x/oauth2 v0.8.0 // indirect
-	golang.org/x/sys v0.13.0 // indirect
-	golang.org/x/term v0.13.0 // indirect
-	golang.org/x/text v0.13.0 // indirect
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.44.0 // indirect
```

Comment:
this probably introduces [CVE-2023-47108](https://github.com/open-telemetry/opentelemetry-go-contrib/security/advisories/GHSA-8pgv-569h-w5rw), 0.46 would be better
(this needs to be fixed upstream too, I wonder why nobody noticed it...)

Reply:
/lgtm cancel

Reply:
> this probably introduces [CVE-2023-47108](https://github.com/open-telemetry/opentelemetry-go-contrib/security/advisories/GHSA-8pgv-569h-w5rw), 0.46 would be better (this needs to be fixed upstream too, I wonder why nobody noticed it...)

Good catch, thank you, @jsafrane ! I added manual bump to 0.46 on top of this PR. Please take a look again.